pkg/gadgets/run/tracer/tracer.go

```diff
@@ -350,6 +354,14 @@ func (t *Tracer) loadeBPFObjects(opts loadingOptions) error {
 		t.topperMap = t.collection.Maps[opts.topperMapName]
 	}
 
+	if histMapName != "" {
+		var ok bool
+		t.histMap, ok = t.collection.Maps[histMapName]
+		if !ok {
+			return fmt.Errorf("missing map named %q", histMapName)
```

Comment:
Also here, if the map is in the spec, it will be in the collection for sure. Otherwise `NewCollectionWithOptions` would have failed.